packages/web/src/components/track-overflow-modal/ConnectedMobileOverflowModal.tsx

```diff
@@ -106,6 +110,12 @@ const ConnectedMobileOverflowModal = ({
   shareUser
 }: ConnectedMobileOverflowModalProps) => {
   const { isEnabled: isEditAlbumsEnabled } = useFlag(FeatureFlags.EDIT_ALBUMS)
+  const { onOpen: openPremiumContentModal } = usePremiumContentPurchaseModal()
+  const openPurchaseModal = useAuthenticatedCallback(
```

Comment:
oo nice.. i probably should have used authenticated callback on mobile as well.

Reply:
I actually started to do the same but then I remembered there isn't even a function for that in mobile since we don't let you go anywhere in app unless you're signed in

Reply:
oh perfect thanks